### DISCUSSION TITLE
How to add custom "more results" link to the end of the hits list?

### DISCUSSION THREAD

**Discussion by mabdelfattah** [algolia/instantsearch#5680] [Q&A] [answered]
I am using *searchBox* widget and limiting the hits to only 5. If the results are more than 5, I want to display “more results” at the end of the hits list, which redirects to a dedicated search page.

**Comment by mabdelfattah** [comment, marked as answer]
connectHits does the job

**Reply by Haroenv** [reply]
connectInfiniteHits or connectpagination likely also would do what you’re looking for :)
